arkane/ess/factoryTest.py

```diff
@@ -69,26 +69,26 @@ def test_determine_qm_software(self):
         terachem_log_path_3 = os.path.join(self.data_path, 'terachem', 'formaldehyde_sp_terachem_results.dat')
         terachem_log_path_4 = os.path.join(self.data_path, 'terachem', 'formaldehyde_coords.xyz')
         terachem_log_path_5 = os.path.join(self.data_path, 'terachem', 'formaldehyde_output.geometry')
-        non_ess_log_path = os.path.join(os.path.dirname(__file__), 'data', 'methoxy.py')
+        non_ess_log_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '../data', 'methoxy.py'))
```

Comment:
I think this is required to pass the tests since I get errors unless I change the directory structure as shown above in green. `factoryTest.py` used to be `utilTest.py` but `utilTest.py` was located one directory higher in `RMG-Py/arkane` while `factoryTest.py` is located in `RMG-Py/arkane/ess`.

Reply:
but `self.data_path` already points to the `../data` level

Reply:
I misread what you were saying earlier my mistake. I changed it to `self.datapath`